lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```diff
@@ -439,10 +439,8 @@ states:
     exitEvents:
       next:
         targetState: KBV_NO_PHOTO_ID
-        targetEntryEvent: next-with-nino
       enhanced-verification:
         targetState: KBV_NO_PHOTO_ID
-        targetEntryEvent: next-with-nino
```

Comment:
Does this need to be `targetEntryEvent: next`? (Is `enhanced-verification` a valid entry event?)

TBH I'm slightly surprised we're mapping this at all - if you have `enhanced-verification` at this point, we shouldn't really route into KBVs, it should just fail. I guess it's largely irrelevant because this should never happen anyway 🤷